universal-login-react/src/ui/UFlow/Devices/Devices.tsx

```diff
@@ -44,6 +45,9 @@ export const Devices = ({walletService, onAccountDeleted, className, basePath =
           className={className}
         />
       </Route>
+      <Route path={`${basePath}/waitingForDeleteAccount`} exact>
```

Comment:
Use `join`, like in other places - `basePath` can have trailing slash, and then there is a problem of `//`

Reply:
👍

Reply:
👍